README.md

```diff
@@ -26,16 +26,44 @@ information [here](https://rosetta-api.org).
 
 ## Features
 * Rosetta API implementation (both Data API and Construction API)
-* UTXO cache for all accounts (accessible using `/account/balance`)
+* UTXO cache for all accounts (accessible using the Rosetta API `/account/balance` function)
```

Comment:
I don't think so you need to mention this function. It's an API, which you already added.

Reply:
Fixing in next commit.